MimeKit/Cryptography/WindowsSecureMimeContext.cs

```diff
@@ -1571,8 +1572,8 @@ public override void Import (X509Crl crl, CancellationToken cancellationToken =
 			if (revoked == null)
 				return;
 
-			foreach (Org.BouncyCastle.X509.X509Certificate certificate in revoked)
-				Import (StoreName.Disallowed, certificate, cancellationToken);
+			foreach (X509CrlEntry crlEntry in revoked)
+				Import (StoreName.Disallowed, crlEntry, cancellationToken);
```

Comment:
TODO: ??

Reply:
The TODO is for the compilation errors: 

Error	CS1503	Argument 2: cannot convert from 'Org.BouncyCastle.X509.X509CrlEntry' to 'System.Security.Cryptography.X509Certificates.X509Certificate2'

Reply:
Seems like there's no way to get the X509Certificate from the CrlEntry and that the X509CrlENtry might not even include them at all, so this might require an entirely new solution.

I wonder how the old BouncyCastle API was able to get the certificate. Maybe it didn't and just created a fake cert with the correct issuer/serial/etc?